.gitignore

```diff
@@ -0,0 +1,29 @@
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
+# Test binary, build with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+### Terraform ###
```

Comment:
curious why considering any terraform related?

Reply:
No reason actually, I copied this file from AWS https://github.com/openshift/cluster-api-provider-aws/blob/master/.gitignore

Reply:
We use to run terraform for some testing there. I don't think we need this now.